htslib/hts_defs.h

```diff
@@ -69,6 +69,18 @@ DEALINGS IN THE SOFTWARE.  */
 #define HTS_DEPRECATED(message)
 #endif
 
+// On mingw the "printf" format type doesn't work.  It needs "gnu_printf"
+// in order to check %lld and %z, otherwise it defaults to checking against
```

Comment:
Please note that %lld should be avoided anyway, use the standard & portable PRIxx constants instead.